README.md

```diff
@@ -1 +1,32 @@
 "# BC-20-Contact-Manger-sms" 
+
+ #This is a contact manager and SMS sender(one way)
+
+ #It stores  contacts in a Sqlite Database.
+
+ #It can also be used to Send SMS to stored contacts.
+
+ #when using for the first time type the following then 
+ press enter:
+
+`npm install`
+ #To add contacts Type the Following:
+
+`node add -n <name> -p <phone number>`
+
+To view Stored contacts Type the following then press enter:
+
+`node view`
+
+#To search for contacts type the following then press #enter:
+
+`node search <name>`
+
+#To delete contacts Type the following then press enter:
+
```

Comment:
The effect you are looking for works when you have a space between the # and the next character. You can also try ##.